app/views/idv/shared/_document_capture.html.erb

```diff
@@ -21,9 +21,7 @@
       endpoint: FeatureManagement.document_capture_async_uploads_enabled? ?
     api_verify_v2_document_capture_url :
     api_verify_images_url,
-      status_endpoint: FeatureManagement.document_capture_async_uploads_enabled? ?
-    send(@step_url, step: :verify_document_status) :
-    nil,
+      status_endpoint: nil,
```

Comment:
Can this be removed entirely ?

Reply:
I'm also generally confused if we're intending to maintain the async implementation, because it seems like it's partly lingering, but this would remove some parts of the implementation toward a less-complete state.